docs/bazel.md

```diff
@@ -156,6 +120,19 @@ Constructs a top-level target for use in `xcodeproj.top_level_targets`.
 A `struct` containing fields for the provided arguments.
 
 
+# Custom Xcode schemes
```

Comment:
In a followup PR the docs for these functions should be adjusted to make more sense on their own, to the user. They should reference `xcode` like `top_level_target1 does.